FEATURE:  GCLOUD DEPLOY IN GKE (#406)
devonfw/hangar: documentation/src/azure-devops/setup-deploy-pipeline.asciidoc

> :provider: Azure Devops

Reviewer: ```suggestion
:provider: Azure DevOps

```

> :pipeline_type: Pipeline

Reviewer: ```suggestion
:pipeline_type: pipeline
```

> :trigger_sentence: This pipeline will be configured to be triggered after the package pipeline

Reviewer: ```suggestion
:trigger_sentence: This pipeline will be configured in order to be triggered every time package pipeline is executed successfully on a commit for `release/*` and `develop` branches, requiring manual launch for other branches but still enforcing that package pipeline has passed
```
